bin/node/runtime/src/lib.rs

```diff
@@ -426,11 +426,16 @@ impl pallet_session::historical::Trait for Runtime {
 	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
 }
 
-const ZeroPercent: Perbill = Perbill::from_percent(0);
-const REWARD_CURVE: PiecewiseLinear<'static> = PiecewiseLinear {
-	points: &[(ZeroPercent, ZeroPercent)],
-	maximum: ZeroPercent,
-};
+pallet_staking_reward_curve::build! {
+	const REWARD_CURVE: PiecewiseLinear<'static> = curve!(
```

Comment:
How did you come up with the numbers?

Reply:
`min_inflation`, `max inflation` and `ideal_stake`  we've selected with @centilex. [Here](https://docs.google.com/spreadsheets/d/1wh_bcCXtE3uK4kg14X9WFcY6VQ72897T9TRBqXyCRl0/edit#gid=971609350) is the detailed calculations.

`falloff` is the same as in Polkadot.

`max_piece_count` - bigger count better quality. It was `40` and we increased to `100` as because we are using the smaller inflation values.
`test_presicion` - we've increased it in comparison with Polkadot. I have [confirmation](https://matrix.to/#/!HzySYSaIhtyWrwiwEV:matrix.org/$164268003514623CHUnf:matrix.parity.io?via=matrix.parity.io&via=corepaper.org&via=matrix.org) from Parity Tech that this is ok.

Reply:
[Here](https://github.com/paritytech/polkadot/blob/594e3b95516693f807f512e0deb26bad2d4f988d/runtime/polkadot/src/lib.rs#L526) you can find Polkadot's parmas.